src/lib.rs

```diff
@@ -1463,6 +1508,253 @@ pub enum KeyIdMethod {
 	Sha512,
 }
 
+/// A certificate revocation list number.
+#[derive(Debug, PartialEq, Eq, Hash, Clone)]
+pub struct CrlNumber {
```

Comment:
Could you use `SerialNumber` instead? This is more or less a copy+rename of that type.

Reply:
Sure, I was on the fence about this one. 5280 describes it as a distinct thing (e.g. not calling it the CRL's serial number) but the specification of encoding is the same as serial so having it distinct in the type system is probably overkill.

One difference I had implemented in the `CrlNumber` type was erroring from the `from_slice` constructor if given something larger than 20 octets. The `SerialNumber` implementation doesn't do that but it seems like a minor point (and could be considered separately, perhaps with other validation).

In any case, I replaced `CrlNumber` with `SerialNumber` :+1: